packages/repository/src/relations/has-many/has-many-through.helpers.ts

```diff
@@ -138,24 +139,33 @@ export function createThroughConstraint<Through extends Entity, ForeignKeyType>(
  *    keyTo: 'productId',
  *  },
  * };
+ * createThroughConstraint(resolvedMetadata, {3, name: 'a product'});
```

Comment:
This isn't a valid JavaScript object initializer, did you perhaps mean the following?

```suggestion
 * createThroughConstraint(resolvedMetadata, {id: 3, name: 'a product'});
```

AFAICT, this example is repeated twice, is that intentional?

Reply:
Also I think you should use `createThroughFkConstraint` name in these examples, not `createThroughConstraint`.

Reply:
Ah I meant to show ` * createThroughConstraint(resolvedMetadata, 3)`, the case that it only take the id value. 

I don't know if it's a good design to make it take both JSON object and the target id. But I think it's redundant to have another helper function to handle this case. WDYT?